### PR TITLE
[alertmanager] add NodePort support

### PR DESCRIPTION
**Pull request by survivant**
added option to support NodePort

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [ X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ X] Chart Version bumped
- [X ] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
chart minor version needs to be updated and commits need to be signed...

**Review by monotek — CHANGES_REQUESTED** [review]
I've tried to fix the linter online but now my commit is not signed.
Could you just fix it yourself and focepush please?

**Comment by survivant** [comment]
> I've tried to fix the linter online but now my commit is not signed.
> Could you just fix it yourself and focepush please?

let me guest.. you try from github edit button.  No idea why they don't put a option to signoff the commit from the UI.  I got stuck there on each project that use DCO :(

**Comment by survivant** [comment]
@monotek  On va l'avoir :)

**Review by naseemkullah — APPROVED** [review]

File: charts/alertmanager/templates/services.yaml [hunk 1]
```diff
@@ -15,6 +15,9 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+      {{- if .Values.service.nodePort }}
```
Comment (naseemkullah) [inline charts/alertmanager/templates/services.yaml:18]:
```suggestion
      {{- if (and (eq .Values.service.type "NodePort") .Values.service.nodePort) }}
```

Does this work?

**Review by naseemkullah — DISMISSED** [review]

File: charts/alertmanager/Chart.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/a
 sources:
   - https://github.com/prometheus/alertmanager
 type: application
-version: 0.8.0
+version: 0.8.1
```
Comment (monotek) [inline charts/alertmanager/Chart.yaml:9]: New feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/alertmanager/Chart.yaml (hunk 2 above)
Comment (survivant) [reply charts/alertmanager/Chart.yaml:9]: @monotek What should I put instead of 0.8.1 ?

Comment (monotek) [reply charts/alertmanager/Chart.yaml:9]: 0.9.0

**Review by monotek — APPROVED** [review]

**Pull request merged**